lab-jaren/app/component/history/history.html

```diff
@@ -0,0 +1,10 @@
+<div class="history">
+  <h3>Your story so far:</h3>
+  <button ng-click="historyCtrl.player.undo()">Undo!</button>
+  <ol>
+    <li ng-repeat="item in historyCtrl.player.history track by item.id">
+      <p>{{ item.desc }}</p>
```

Comment:
Per the bug that Duncan realized today during code review, let's make our ``ng-repeat`` do a ``track by $index`` now.